CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * Added the `ReplayLocationManager.replayCompletionHandler` property that allows you to loop location. ([#3528](https://github.com/mapbox/mapbox-navigation-ios/pull/3528), [3550](https://github.com/mapbox/mapbox-navigation-ios/pull/3550))
 * Changed the behavior of `ReplayLocationManager` so that it doesn't loop locations by default. ([#3550](https://github.com/mapbox/mapbox-navigation-ios/pull/3550))
 * Fixed an issue where `ReplayLocationManager` didn't update location timestamps when a new loop started. ([#3550](https://github.com/mapbox/mapbox-navigation-ios/pull/3550))
+* Fixed an issue where `UserPuckCourseView` is trimmed when using custom frame for `NavigationMapView.userLocationStyle`. ([#3601](https://github.com/mapbox/mapbox-navigation-ios/pull/3601))
```

Comment:
```suggestion
* Fixed an issue where `UserPuckCourseView` is trimmed when using custom frame for `UserLocationStyle.courseView(_:)`. ([#3601](https://github.com/mapbox/mapbox-navigation-ios/pull/3601))
```

Reply:
Thanks for the suggestion, I've updated the changeLog.